src/convertToObject.js

```diff
@@ -1,12 +1,30 @@
 'use strict';
 
+const toCamelCase = require('./toCamelCase');
+
 /**
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const splitString = sourceString.split(';');
```

Comment:
The variable name `splitString` is not descriptive of its contents. It contains CSS rules, so a more descriptive name like `cssRules` would improve readability.